EJ2-Documentation-853041-Ug-Documentation-For-Row-Selection

harinikrh opened the pull request.

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [EJ2-grid-Row-Selection](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/148/console)**

SyncfusionBuild commented:
**CI Status: SUCCESS :white_check_mark:<br/>Source Compilation: SUCCESS :white_check_mark:<br/>Build Location: [EJ2-grid-Row-Selection](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/148/)<br/>**

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [EJ2-grid-Row-Selection](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/151/console)**

SyncfusionBuild commented:
**CI Status: SUCCESS :white_check_mark:<br/>Source Compilation: SUCCESS :white_check_mark:<br/>Build Location: [EJ2-grid-Row-Selection](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/151/)<br/>**

SyncfusionBuild commented:
**Build Status: INQUEUE :clock3:<br/>Build Location: [EJ2-grid-Row-Selection](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/154/console)**

SyncfusionBuild commented:
**CI Status: SUCCESS :white_check_mark:<br/>Source Compilation: SUCCESS :white_check_mark:<br/>Build Location: [EJ2-grid-Row-Selection](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/154/)<br/>**

VigneshNatarajan27 reviewed (changes requested):
selectColumns API links is provided instead of selectRows API link


Select grid rows based on certain conditions topic, Javascript platform name is not properly displayed

SyncfusionBuild commented:
**Build Status: INQUEUE :clock3:<br/>Build Location: [EJ2-grid-Row-Selection](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/184/console)**

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [EJ2-grid-Row-Selection](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/184/console)**

SyncfusionBuild commented:
**CI Status: SUCCESS :white_check_mark:<br/>Source Compilation: SUCCESS :white_check_mark:<br/>Build Location: [EJ2-grid-Row-Selection](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/184/)<br/>**

harinikrh commented:
> selectColumns API links is provided instead of selectRows API link
> 
> Select grid rows based on certain conditions topic, Javascript platform name is not properly displayed

Changed the API link for selectRows
Changed in to platform name

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [EJ2-grid-Row-Selection](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/185/console)**

SyncfusionBuild commented:
**CI Status: SUCCESS :white_check_mark:<br/>Source Compilation: SUCCESS :white_check_mark:<br/>Build Location: [EJ2-grid-Row-Selection](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/185/)<br/>**

VigneshNatarajan27 reviewed (changes requested):
persistSelection spelling is wrong

MaithiliyK commented on ej2-javascript/code-snippet/grid/grid-cs184/index.js:
```diff
@@ -0,0 +1,66 @@
+});
+grid.appendTo('#Grid');
+
+var mod;
```
always I insist to provide meaningful name

harinikrh replied:
changed the variable name

MaithiliyK commented on ej2-javascript/code-snippet/grid/grid-cs184/index.js:
```diff
@@ -0,0 +1,66 @@
+  else {
+    grid.pagerModule.goToPage(page);
+    if (grid.pagerModule.pagerObj.currentPage == page) {
+      grid.selectRow.mod;
```
is this code is correct. grid.SelectRow.mod is not a proper code check on it

harinikrh replied:
changed in to grid.SelectRow(mod)

MaithiliyK commented on ej2-javascript/code-snippet/grid/grid-cs185/index.js:
```diff
@@ -0,0 +1,24 @@
+});
+grid.appendTo('#Grid');
+
+var enableSelection = false;
```
is this variable needed here

harinikrh replied:
Removed the variable

MaithiliyK commented on ej2-javascript/code-snippet/grid/grid-cs185/index.ts:
```diff
@@ -0,0 +1,30 @@
+});
+grid.appendTo('#Grid');
+
+let enableSelection = false;
```
is the variable needed here

harinikrh replied:
Removed the variable

MaithiliyK reviewed (commented):
address my review comments

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [EJ2-grid-Row-Selection](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/190/console)**

VigneshNatarajan27 reviewed (approved):
changes are fine

SyncfusionBuild commented:
**CI Status: SUCCESS :white_check_mark:<br/>Source Compilation: SUCCESS :white_check_mark:<br/>Build Location: [EJ2-grid-Row-Selection](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/190/)<br/>Image Alt Text Error(s): [0](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/190/artifact/cireports/ImageValidation/AltTextValidation.html)<br/>Image Size Error(s): [0](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/190/artifact/cireports/ImageValidation/ImageSizeValidation.html)<br/>Image Name Error(s): [0](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/190/artifact/cireports/ImageValidation/ImageNameValidation.html)<br/>FT Layout Syntax Error(s): [0](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/190/artifact/cireports/FTLayoutSyntaxValidation/FTStructureValidation.html)<br/>Front matter Error(s): [0](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/190/artifact/cireports/FrontMatterValidation/FrontMatterValidation.html)<br/>File Path Error(s): [0](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/190/artifact/cireports/FilePathValidation/FilePathValidation.html)<br/>CodeSnippet File Path Missing Error(s): [0](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/190/artifact/cireports/CodeSnippetFilePathValidation/CodeSnippetFilePathValidation.html)<br/>Toc Anchor Tags Validation Error(s): [0](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/190/artifact/cireports/AnchorTagsValidation/AnchorTagsValidation.html)<br/>**

SyncfusionBuild commented:
**Build Status: INPROGRESS :arrows_clockwise:<br/>Build Location: [EJ2-grid-Row-Selection](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/191/console)**

SyncfusionBuild commented:
**CI Status: SUCCESS :white_check_mark:<br/>Source Compilation: SUCCESS :white_check_mark:<br/>Build Location: [EJ2-grid-Row-Selection](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/191/)<br/>Image Alt Text Error(s): [0](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/191/artifact/cireports/ImageValidation/AltTextValidation.html)<br/>Image Size Error(s): [0](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/191/artifact/cireports/ImageValidation/ImageSizeValidation.html)<br/>Image Name Error(s): [0](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/191/artifact/cireports/ImageValidation/ImageNameValidation.html)<br/>FT Layout Syntax Error(s): [0](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/191/artifact/cireports/FTLayoutSyntaxValidation/FTStructureValidation.html)<br/>Front matter Error(s): [0](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/191/artifact/cireports/FrontMatterValidation/FrontMatterValidation.html)<br/>File Path Error(s): [0](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/191/artifact/cireports/FilePathValidation/FilePathValidation.html)<br/>CodeSnippet File Path Missing Error(s): [0](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/191/artifact/cireports/CodeSnippetFilePathValidation/CodeSnippetFilePathValidation.html)<br/>Toc Anchor Tags Validation Error(s): [0](https://ci.syncfusion.com/job/Content/job/ej2-grid-docs/191/artifact/cireports/AnchorTagsValidation/AnchorTagsValidation.html)<br/>**

MaithiliyK reviewed (approved):
changes are fine

VigneshNatarajan27 reviewed (commented):
changes are fine. since it is duplicate of already reviewed PR
